docker/production.yaml

```diff
@@ -0,0 +1,55 @@
+listen:
+  port: 9000
+
+# Correspond to your reverse proxy "listen" configuration
+webserver:
+  https: true
+  hostname: undefined
+  port: 443
+
+# Your database name will be "peertube"+database.suffix
+database:
+  hostname: 'db'
+  port: 5432
+  suffix: '_prod'
+  username: 'postgres'
+  password: 'postgres'
+
+# From the project root directory
+storage:
```

Comment:
If these are relative path, then the volume at ``/data`` will not contain them. Thus, no data will be persistent.